cmake/configs/aarch64/linux/config.cmake

```diff
@@ -6,7 +6,7 @@ set(WT_OS "linux" CACHE STRING "")
 set(WT_POSIX ON CACHE BOOL "")
 
 # Linux requires '_GNU_SOURCE' to be defined for access to GNU/Linux extension functions
-# e.g. Access to O_DIRECT on Linux. Append this macro to our compiler flags for Linux-based
+# e.g. Access to 'pthread_setname_np' on Linux. Append this macro to our compiler flags for Linux-based
```

Comment:
I think there is an opportunity here to refactor the multiple places this is required into a single location using cmake [CheckSymbolExists](https://cmake.org/cmake/help/latest/module/CheckSymbolExists.html). 
Thoughts on creating a ticket?

Reply:
I created [WT-13259](https://jira.mongodb.org/browse/WT-13259) to capture this.